src/Admin/AbstractAdmin.php

```diff
@@ -2073,7 +2090,7 @@ public function getRouteBuilder(): ?RouteBuilderInterface
         return $this->routeBuilder;
     }
 
-    public function toString(object $object): string
+    public function toString(?object $object): string
```

Comment:
```suggestion
    public function toString(object $object): string
```

ToString should not allow null as param.